In [2]:
from Notes import *

Render_Notes('A')

[NbConvertApp] Converting notebook file_to_convert.ipynb to pdf
/home/taylor/.local/lib/python3.8/site-packages/nbconvert/utils/pandoc.py:51: RuntimeWarning: You are using an unsupported version of pandoc (2.5).
Your version must be at least (2.14.2) but less than (4.0.0).
Refer to https://pandoc.org/installing.html.
Continuing with doubts...
  check_pandoc_version()
[NbConvertApp] Writing 52239 bytes to notebook.tex
[NbConvertApp] Building PDF
[NbConvertApp] Running xelatex 3 times: ['xelatex', 'notebook.tex', '-quiet']
[NbConvertApp] Running bibtex 1 time: ['bibtex', 'notebook']
[NbConvertApp] WARNING | bibtex had problems, most likely because there were no citations
[NbConvertApp] PDF successfully created
[NbConvertApp] Writing 74828 bytes to Notes_A.pdf


- Welcome to Intermediate Spanish
- Who am I?
- Hype the class
- Syllabus
- Episode A0


## Who am I?


I'm Professor Taylor Weidman. You can call me Taylor. Even though I can run a 5 minute mile, I'd still prefer you don't call me Taylor Swift. 

I've been in the econ department at Pitt for 7 years. And I've been teaching microeconomics for nearly as long.

I was born in Arlington VA. Anyone from the DMV?

I grew up in Richmond. Anyone from Richmond?

We then moved to south central PA. Anyone from that area?

I did undergrad at a small school in the Shenandoah Valley, did a masters in economics at Iowa State, then moved here for the PhD in 2016. 


## Hype the class

Every semester, when teaching assignments are being decided, I jockey for position to teach this class.

When I was in undergrad, I took this course in my first year, and it shaped my worldview more than probably any other. Who here is a first year? 

There are a couple of lightbulb moments I'm excited to show you. 

I get so excited to walk though these ideas with you all. I've been in the field so long, but its really rewarding for me to get to remember why I fell in love with this field in a room full of bright minds.


## Class Logistics

I do a few things differently than other classes. 

In lecture, we talk about ideas, work with these ideas, and develop the skills to understand the ideas. 

Lectures will be a mixture of animations, simulations, chalkboard work, and group work.

I try to avoid slides. I don't like presenting with them. And I didn't like learning from them as a student. 

I think you'll have a better time learning without slides. But there are two downsides. First, its more work for me. I have to do extra to teach with a chalkboard. Second, you can't just print off slides. You have to take notes, writing down what you think is most important. But doing class this way is just better, according to the research.

Recitations are equivalent to a lab. You get to work with a trained economist in a more informal setting.

Recitations extend the work we do in this room, and help you prepare for MiniExams.


## Grades

I'll be upfront about my views on grades. In an ideal world we wouldn't have grades. 

But the university likes them. And they give you incentives to be excellent. 

The point I try to make here is that my main hope for you is that you'll pursue your curiosity and try to be excellent. So I design grades to make that possible.

Talk to me if you ever think any part of the class isn't working to make that as easy as possible for you. 

I work hard to give my students all the resources to succeed.

My goal is to show you why these ideas have been some of the most influential in modern history and walk you though the skills required to understand them. 

*syllabus and website*

This class isn't easy. But it's set up to give each of you the resources to do as well as you're hoping. 

I don't curve this class. Everyone can get an A, and I'd be thrilled. You just have to hit the bar.

I ask that you respect the work I put into this class. And in return I'll give you the respect you deserve as a bright student at an excellent university. 


## Talking to Me

One resource available to you is office hours. Come talk to me. 

When I was a student, and even to some degree in grad school, I was often intimidated or nervous to talk to my professors. 

You don't have to be that way with me. I'm an expert in this field. But you can come talk to me. 

You don't know that yet. So to try to prove it to you, let me read some things my students have said in the past. 

- Taylor is willing to help and explain topics.
- Taylor is very interested in the course and wants his students to succeed.
- Taylor understands his students and makes learning non-stressful.
- Taylor is very approachable and makes class interesting. 

Come talk to me! It's one of my favorite parts of my job.


## Economics isn't *about* money | Part A | Episode 0

Often when I hear about economics in popular culture or on the news it has to do with prices, stocks, market indicators, unemployment, maybe with a ticker tracking something financial. It can be important to communicate these things. Having some sense of how and whether people are able to get jobs gave us one view of one of the impacts of the Great Recession. And then again when Covid hit.

***Show a time series of unemployment starting in the early 2000s***

These economic measures communicate some things to some people. Economics at base, however, isn't *about* money or unemployment. 

Lets start at the beginning.


### Economics is about decisionmaking. 

### Preferences

Each of us has preferences. I prefer carrot cake to chocolate cake. 

***Show two cakes with a preference symbol***

I know I know ... send hate mail to econ@hatemail.org. I also prefer dark roast coffee to medium roast coffee. 

***Show two coffees with a preference symbol***

But I prefer medium roast coffee to light roast coffee.

***Add light roast coffee with a preference symbol to the end of the string***

I could keep going. I have preferences for most things in my life. And if you're human, you do too. I could ask you about most things and you'd be able to tell me which you prefer. You'd be able to rank them. All we have to do is list the choices from best to worst, assign them numbers, with larger numbers being more preferred, and this ordering represents our preferences. 

This is how economists think about preferences. 

***Show "Preferences are rankings."***


## Scarcity

But sorry if I'm the one to break it to you, we can't always have what we want most. 

***Show "We can't always have what we want most.***

Most people prefer a Tesla Model Y to a Honda Civic. But for many reasons not everyone right now can have a Model Y. And while we face financial constraints it's important to realize that scarcity like this is more basic than money. 

***Show "Scarcity is more basic than money."***

Even during the times in history when we didn't have money, human beings were still unable to have everything they wanted. 

There's always been tension between getting one thing or getting another. This happens both at an individual level and at a societal level. If we decided everyone should have a Tesla, it might be physically possible, but in doing so someone may go hungry. 

***Show "Everyone Gets a Tesla vs Everyone Gets Food"***

Society faces all sorts of choices like how many Teslas to produce and who should get them. And this decision-making is necessary precisely because we have preferences AND we face scarcity. With preferences in the face of scarcity, we must make choices requiring tradeoffs. 

***Show "With preferences in the face of scarcity, we must make choices requiring tradeoffs."***


## Preferences + Scarcity = Choices


Something about how scarcity tells us what's possible and then we can pick which possible thing we like the best.

***Preferences + Scarcity = Choices***

This is the world economics lives in. Given that we are making choices in scarcity to maximize preferences, what should we do?


## Tradeoffs


To get a better handle on choices, lets be a bit more sophisticated how we define tradeoffs. Lets say I give you the choice of $A$ OR $B$. If you choose $A$, you can't have $B$. If you choose $B$, you can't have $A$. Economists define opportunity cost as the value of the next best alternative. This definition gives us handy conceptual traction here. If you choose $A$, your next best alternative is $B$. So your opportunity cost of $A$ is $B$. And your opportunity cost of $B$ is $A$. We’re going to use opportunity cost quite heavily as we build more sophisticated models. 


### Economics is about social environments. 


## Economics is about what's possible | Part A | Episode 1

## Molly's Opportunity Cost

Let's build up some intuition for opportunity cost with a little example. Imagine Molly, a small scale organic farmer, is good at growing spinach and carrots on her $100$ hectare farm. If she spends all her time growing spinach she can grow $45$ tons per year. If instead she spends all her time growing carrots, she can grow $5$ tons per year. If Molly were to switch from growing all carrots to all spinach, she would give up $5$ tons of carrots and instead get $45$ tons of spinach. So we’ll set those two values equal and simplify. And that's it. Molly’s opportunity cost of $1$ unit of carrots is $9$ units of spinach. For every $1$ unit of carrots she wants to grow, she’ll have to give up $9$ units of spinach. This idea is crucial to what we'll do later.


## Molly's Production Possibility Frontier

There is a more systematic way to think about Molly’s farm. We've gone through and calculated Molly's opportunity cost for her farm. But Molly doesn't need to not choose either carrots OR spinach; she can choose any combination of the two crops. So let's go through and examine a range of crop choices she can make. Let's use an $x,y$ graph to plot how much of each crop model is harvesting as we move through a few crop choices. Let's use the vertical axis to represent her carrot harvest and the horizontal axis to represent her spinach harvest. We've already established that she can harvest $5$ tons of carrots and $0$ tons of spinach, or $0$ tons of carrots and $45$ tons of spinach. 


Based on her productivity numbers, if Molly plants carrots on half her land and spinach on the other half, she would have half of $5$ tons of carrots and half of $45$ tons of spinach. Let's also plot that on our $x,y$ graph.


At this point you might wonder how Molly would choose how much of each to plant. Great question. We'll come back to it in later chapters. For now we're just gonna find what's possible to set up the trade-offs involved in our later models.


We can pick another point, splitting the land into $\frac{3}{5}$ carrots and $\frac{2}{5}$ spinach.


As we running through more points its easy to see that any carrot and spinach combination Molly chooses lies on a line which we call the Production Possibility Frontier (PPF). Take a look at the range of other combinations. And guess what? Calculating the the slope of this relationship gives us her opportunity cost. To harvest $1$ more unit of carrots Molly must havest $9$ fewer units of spinach. The slope here is negative since we're giving one good up for the other, exactly what opportunity cost is capturing. This is really the trade-off captured by the opportunity cost and as we mentioned earlier it's embedded in the PPF relationship.


## In Closing

This relationship is what one person can do on their own. If this were the best one could do, the story would be pretty boring. And we'd be pretty close to finished with this class.

But some foundational insights in the 1800s give us our first counter-intuitive punchline: we can actually do better!

Showing how we do this requires some mathematics and a small detour into 1800s economic history. And that's exactly where we're going in the next few classes. This is Part A of the class, where we set up the landscape of everything that comes later in the semester.

## Coordination

Most of the time we make choices that impact others. One day you might find yourself stranded by yourself on a desert island or Martian moon. But if everything is going to plan I’d imagine you’re not there right now. And just like me, when you need food you need other people. This interaction, precisely how my choices interact with the choices of others when there are stakes, that’s what economics considers. 


There are a few main tools we use socially to coordinate our choices. Money IS a big one. Prices at their best tell us something about the relative value of something. If I give you the option to buy a carrot for a dollar, you’ll consider how much you’d like the carrot and how much you value the dollar and it's other uses. And when you choose the one you like the most you’re saying something about their relative value to you in that moment. 

And in democracies, votes are another important tool. Given the choice between two mayors, I can choose the one with the tax policy or reform agenda I like the best. But I can't have both mayors at the same time. 

Coordination devices make these types of things possible. 


That’s the core of economics: decision making and coordination with tradeoffs. And there’s a bit of magic ahead, both in what coordination can do for us, but also in pointing out social dilemmas, where coordination can and often fails catastrophically. 

I entered economics thinking about how to solve climate change. If you’re at all interested in these types of questions, you’re in the right place. These videos are aimed at folks of all ages. Critical and numerate thinking is helpful but you don’t need a background in economics, math, or statistics.

This first series sets the stage for where we’re going, introducing a toy world we’ll play with in the videos to come. And the basic question to keep in mind is: how can we use the idea of trade offs to make those involved better off? 


You'll notice both preferences and scarcity aren't directly about money or votes. We'll use both as tools, but the tools aren't the landscape. 

# More *can* be possible! | Part A | Episode 2

## Notes
The last part with finding the trade should be its own video. Students are having a tough time with the visuals. And I'm having a tough time explaining them in a way that works.


## The Set Of Possibilities

**SKIP** You'll notice these new harvests lie *inside* the PPF, not on the frontier. This is what we call ***inefficient***, since there's productive land that's gone on used. You might be thinking that maybe she had a good reason to leave it unused. And yes sleeping is certainly important. And we'll get to how Molly chooses how much of each crop to plant later in this semester. But just for the moment all we can say is that there is missed potential harvest. 


It turns out Molly has been working $10$ hours per day, but realizes she's burnt out and needs to spend less time working. She decides to spend only $8$ hours per day on her farm, which means she is now only able to grow on $\frac{4}{5}$ of her $100$ hectare farm. With this new work schedule she can only farm $80$ hectares. Her productive capacity has dropped to either $4$ carrots or $36$ spinach. 


And this is fine if this is her preference and she can still make her ends meet. The rest of the points along the PPF have shifted in with the extreme points. At this point I would recommend convincing yourself that any combination of crops lies on this new PPF. 


This may seem like bad news for Molly’s wallet. And it could have been. But it turns out that later that winter she comes across a new carrot harvesting technology that would double her carrot harvest. With this technology she's able to grow up to $8$ carrots. This turns out to pivot the PPF out on the carrot axis.


So what’s happened here? A change in Molly’s labor input has changed the output of her farm. The frontier depends on the level of labor. The more labor, the more is possible. And in this case since there's less labor, less is possible. And in addition to labor, the PPF also captures changes in technology. These changes in the PPF aren’t purely mathematical. What we’ve found is a region that used to be inefficient that is now unattainable. And a region that used to be unatainable that is not on the fronteir.


Molly’s PPF depends on how much labor goes into her farm, and the efficiency of her technology. Which means that her PPF shifts in response to changes in labor and technology. But to do this, inputs had to change. But if this discussion leaves you wondering if there’s some way to exceed the PPF without changing inputs, using only our choices, you're probably in the right class. And of course as it turns out there’s a more profound insight, a way to exceed the frontier without changing inputs or technology. But it requires some more math and a journey into 1800s Britain. 


## Guilding

It turns out Molly isn’t the only farmer in the area. Another farmer, Andrew, operates another small scale organic farm next to Molly’s. He also is good at growing carrots and spinach. He can grow $4$ tons of carrots per year and $18$ tons of spinach per year on a similar $100$ hectare farm. Molly is also able to produce more of both goods, so she has the ***absolute advantage*** in both carrots and spinach. 


Molly and Andrew decide to form a local farmers guild, operating their respective farms but shipping their produce together. Let's say the guild decides that both farms will produce the same ratio of carrots and spinach. Dividing their farms equally between crops, Andrew will produce $2$ carrots and $9$ spinach, and Molly will produce $2.5$ carrots and $22.5$ spinach. This adds up to $4.5$ carrots and $31.5$ spinach. Under this regime, the guild's PPF is just a line running through the guild's extreme points $9$ carrots and $63$ spinach, just the extreme points for the two farms added together.


But Molly, having studied economics in college realizes that opportunity cost might be important to consider when choosing their crops. Like we did for Molly, Andrew’s opportunity cost of carrots is $\frac{9}{2}$ spinach. Again, this is visualized by Andrew's steeper PPF, meaning he gives up more carrots for each ton of spinach relative to Molly.


So Molly has a lower opportunity cost of carrots and Andrew has a lower opportunity cost of spinach. Molly has to give up $\frac{1}{9}$ units of carrots for every $1$ unit of spinach, and Andrew has to give up $\frac{2}{9}$ units of carrots for every $1$ unit of spinach. So Andrew is relatively less efficient at growing spinach because he’s giving up more carrots to grow it. And Andrew, therefore, is relatively more efficient at growing carrots since he's giving up less spinach to grow it. This is the flip side of the opportunity cost equation we used at the beginning of the video. Andrew's next best use of his land is less costly when growing carrots even though Molly is better at growing carrots. This is because she is *much* better at growing spinach than Andrew. This is one of those important and somewhat unintuitive idea's you'll run into periodically in economics.


Maybe pause here and convince yourself that even though Molly is better at growing both goods, Andrew has a lower opportunity cost of growing carrots than Molly. He is gives up less than she does to grow carrots.


## Comparative Advantage

Having a lower opportunity cost is what we call **comparative advantage**. And here, Andrew has a comparative advantage in carrots, and Molly has a comparative advantage in spinach. As you might have seen before, there is an important mathematical realization here. When one farmer’s opportunity cost is lower in one crop, the other farmer’s opportunity cost will always be lower in the other.


So Molly realizes that maybe there’s something to this idea, that maybe the guild should be minimizing their joint opportunity cost. So she and Andrew decide to specialize in what each farmer is relatively most efficient at growing, what they have a comparative advantage in. So Molly grows only spinach, growing $45$. And Andrew grows only carrots, growing $4$.


And we can check our work here by checking what happens to our production when we specialize in the good we don't have the comparative advantage in. And our production point is inside the guild's equal-split PPF, so we've correctly chosen the specialization according to comparative advantage. 


So without changing any technology, simply by choosing their crop according to their comparative advantage, the local farmers guild was able to exceed the frontier. Its a more general truth that so long as we're not too close to the edges, we can exceed any point on the PPF with specialization. The limit to this is when one farmer produces all of the good they don't have a comparative advantage in.


Why is this true? We’ve found the tradeoffs underlying the farmers’ crop choices, and we chose crops that minimize costs. Andrew has a lower opportunity cost of carrots. He gives up less to grow carrots, and Molly gives up less to grow spinach. So specializing in the lower opportunity cost crop makes the guild do better than a simple split.


This tells us something deep about opportunity cost, that by making choices together that minimize opportunity cost can make Molly and Andrew better off jointly. But possibly you're still concerned that the GROUP is better off but one of the INDIVIDUALS in the group may not be. We were slightly concerned that maybe we were sacrificing the individual for the benefit of the group. So let's turn to the model proposed by David Ricardo in 1817 to explain how specialization and trade can make two countries better off. 


## Coordination Using Prices / Specialization and Trade

So far we’ve looked at a type of cooperation between two individuals. Here the setup here is a little different, where instead of forming a guild together, they both aim to maximize their own harvest individually. This is where the real magic happens. So to model this environment, we'll set up two PPFs one for Molly and one for Andrew. 


Production for the two farms together in a guild was better off, but could we exceed the frontier of both farms individually at the same time? Is there some coordination device that would make this possible? The answer here again is YES!


In autarky the farmers don't trade with each other and, like before, these PPFs represent the frontier for each farm. In autarky, then, Molly can grow $5$ tons of carrots OR $45$ tons of spinach while Andrew can grow $4$ tons of carrots OR $18$ tons of spinach. Lets pick a point in autarky and see if we can do better with trade, say $1$ ton of carrots and $36$ tons of spinach for Molly, and $3$ tons of carrots and $7$ tons of spinach for Andrew.


So what if we realized that since we could exceed the joint PPF from last time, we wondered if we could make both farms better off simultaneously. We have a hunch that since minimizing opportunity cost by specializing according to comparative advantage improves the whole, that we might be able to follow a similar program and improve both farmers. But additionally, to give both farmers access to both crops, we also need to trade.


The first thing to realize when we're thinking about trading is that each farmer can perform a sort of trade with themself. Each farmer could, if they wanted to, switch part of their farm from the crop they're growing under specialization to the other good. Thought of as a trade, this exchange is exactly equal to their opportunity cost, and is what the PPF tells us. So we're looking for any trade that is more favorable than their PPF. We can represent this with a line. As long as the trade lives outside the farmer's PPF, the trade benefits them. 


To visualize this trade we just need to realize that when we give up the good that we're specializing in, we also receive the other good in exchange. This lets us view the trade as a downward sloping relationship between the two goods, where the slope represents the exchange rate. And this is going to look very similar to a PPF except it's not going to intercept our two extremes. It's just going to run through the point where we're currently producing.


**SKIP** If the trade line lies below the PPF, the trade doesn't do better than the PPF for the farmer. So we'll only highlight the acheivable area made possible by the trade. We also need to realize that not every trade is feasible, though, since the other farmer can only away what they're harvesting. So we constrain the trade line on its upper end as well. This leaves us with a line representing the PPF as well as a line representing the range of possible trades.


If the trade line lies below the PPF, the trade doesn't do better than the PPF for the farmer. So we'll only highlight the acheivable area made possible by the trade. We also need to realize that not every trade is feasible, though, since the other farmer can only away what they're harvesting. So we constrain the trade line on its upper end as well. This leaves us with a line representing the PPF as well as a line representing the range of possible trades.When thinking about trades, we have two levers. First, the exchange rate, represented by the slope of the trade line. If the trade is too steep, its great for one farmer and terrible the other. And if its too flat, the same but reversed. So the only trades that are acceptible happen when the exchange rate lies between the opportunity costs of the two farmers.


And second, we can choose how much of each goods the farmers grow. Earlier we showed that Molly has a comparative advantage and spinach and Andrew has a comparative advantage in carrots so each is gonna specialize in each of these two goods. Each farmer specializing according to their comparative advantage, Molly harvests $45$ tons of spinach and Andrew harvests $4$ units of carrots. This is like the guild solution, but here we're looking for a trade that makes both farmers better off. And it's important to note here that we STILL haven't imposed preferences. All we're looking at is the frontier, and asking if we can find a way to live outside it without changing inputs.


As Molly shifts toward her specialty crop, she's able to trade more of her speciaty to Andrew, and Andrew is able to exchange for more of the good not his specialty. But the real magic happens when we move both Molly AND Andrew toward their specialty's, extending the range of possible trades.

At this point it becomes clarifying when we plot the trade lines on one graph for the farmers together.

Returning to the challenge we set out on, specializing, exchanging $C$ carrot for $S$ spinach sets the exchange rate to $X$, which works for both farmers, means both farmers have more of both goods than they had in autarky.

The benefits of trade become more extreme if opportunity cost were greater for either or both of the farmers.

It turns out in both goods both farmers are doing better than they were before. So not only have we improved the group as a whole but we've also improved each individual farmer by choosing the appropriate exchange rate. This isn’t some mathematical sleight of hand. No matter what the original production levels were, as long as both farmers have different productive capabilities, we can always do better for a range of points on the PPF with specialization and trade. We’ve chosen the original values to make the math work out cleanly. But there’s nothing special about them.


One question I'll leave you with: can we always find a trade like this?

We’ve done something extraordinary here. If this doesn’t spark your interest in the field, I’m not sure I can help you! We’ve shown that two types of coordination, guilding and trading, not only increase the total harvest, but also can increase the crops for each farmer individually. And we did this simply by realizing that tradeoffs are important, and that exchanges that take advantage of opportunity cost and comparative advantage mean cooperation can make both farmers better off without giving up any harvest in the process. We’ve just put numbers to something that at the outset might have sounded impossible.


Interaction between decisionmakers like this is central to what we do in microeconomics. Welcome! We have a lot to do.

# The Landscape | Episode 3 | Part A

This video relates a lot of ideas together, basically saying: Isn't this cool! By the end of this thing you'll understand what's going on.

# Transcript

This is the tutorial video where I set the stage with the Four Quadrant PPF, and say something like "this probably won't make sense yet. by the end of this series you'll have an idea for what's going on".

The idea is to link together the idea of the PPF, how production functions inform the firm's choice over which type of thing to do, how that choice informs the labor market and prices there, how markets answer the question of where on the PPF to live, how the preferences in the edgeworth box give us prices and answer the question of who gets what, how the prices have to be the same, how the PPF comes from the edgeworth production box. etc

The idea is to zoom way out to see the forest, then zoom back into each part. Even do this in the video. Start by talking about the types of things that could change, show them changing and what that means for each part. But then zoom into each part separately, to isolate one part without the complexity. Then zoom back out, and zoom into another part, kind of following the thing around. I think this could be a really fun video.

## The Map



Today is the first day of the rest of the class. 

Last week we began the journey into microeconomic theory where economics all began. Scarcity constrains our ability to meet all our wants which requires us to make choices. We found the definition of the outer edge of our constraints, the PPF, and that choices along the frontier require tradeoffs. Then when we formally defined the cost of each tradeoff as opportunity cost we realized we could minimize what we gave up by specializing and trading, effectively increasing what is attainable. This is the magic that accompanies clear economic thinking.

During our discussion of these foundational limitations at the heart of economics, we conveniently avoided saying anything too emphatic about WHERE we would like to be on the frontier. All we said is that the frontier is where we would like to be, and that opportunity cost is at the heart of how we conceive of tradeoffs. With all this in mind, we’re ready to begin thinking about how we think about where on the frontier we would like to be. 

This is a highly non-linear map of the outlines of microeconomics. 
